stdlib/socket/src/main/ballerina/socket/Module.md

```diff
@@ -0,0 +1,63 @@
+## Module overview
+This module provides an implementation for connecting to remote socket server or act as a server for incoming socket request. The module facilitates two types of endpoints as `Client` and `Listener`.
+## Samples
+### Listener endpoints
+The sample given below shows how a listener is used to listen to the incoming socket request. The `onAccept(socket:Caller)` resource function gets invoked when a new client connected. New client will be represented using the `socket:Caller`.
+`onReadReady(socket:Caller, byte[])` resource will get invoked once the remote client sends some data and `onClose(socket:Caller)` will invoke once the client is departed. 
```

Comment:
Let's change this to "`onReadReady(socket:Caller, byte[])` resource gets invoked once the remote client sends some data and `onClose(socket:Caller)` is invoked once the client departs."

Let's avoid future tense as much as possible. Normally if the word "will" is in a sentence it is indicative of future tense. Use simple present instead. Apply elsewhere if applicable.